bin/jira/track_flaky_tests.sh

```diff
@@ -17,6 +17,7 @@ for TEST in "${TESTS[@]}"; do
   TEST_CLASS=$(xmlstarlet sel --template --value-of '/testsuite/testcase/@classname' ${TEST})
   TEST_NAME=$(xmlstarlet sel --template --value-of '/testsuite/testcase/@name' ${TEST})
   TEST_NAME=${TEST_NAME% (Flaky Test)}
+  TEST_NAME=${TEST_NAME%%[*}
```

Comment:
What does this do?

Reply:
It removes the number of execution attached to the test name, rest API doesn't like it:
```
ISSUES='{"errorMessages":["The text query '\''Flaky test: org.infinispan.commons.util.MutableIntSetTest#testFlaky1[1]'\'' for field '\''summary'\'' is not valid: probably your range query is incorrect."],"errors":{}}'
```

Reply:
Any idea why this isn't an issue with Jenkins?

Reply:
tbh no, let me see

Reply:
found
currently the flaky script doesn't run due to a missing space here before the command:
https://github.com/infinispan/infinispan/blob/4f2c6dd6a2cd4a718ecd1206d0f3320be5b03b46/Jenkinsfile#L208

also TARGET_BRANCH was empty

fixed those API fails for the brackets, see https://ci.infinispan.org/job/Infinispan/view/change-requests/job/PR-12596/9/consoleFull

Reply:
Thanks for looking  into this! It's bad that we haven't been reporting tests since March and even worse that we didn't notice :scream: